schema.yaml

```diff
@@ -57,41 +60,68 @@ components:
           title: CPU Limit
           type: string
           description: CPU limit. e.g. 0.5, 1, 2
-          default: ''
+          default: ""
           x-walrus-ui:
             group: Resources
         limit_memory:
           title: Memory Limit
           type: string
           description: Memory limit. e.g. 128Mi, 512Mi, 1Gi, 2Gi, 4Gi
-          default: ''
+          default: ""
           x-walrus-ui:
             group: Resources
         namespace:
           title: Namespace
           type: string
           description: Namespace to deploy. Auto-generated if empty.
-          default: ''
+          default: ""
           x-walrus-ui:
             group: Advanced
         name:
           title: Deployment Name
           type: string
           description: Name of the deployment resource. Auto-generated if empty.
-          default: ''
+          default: ""
           x-walrus-ui:
             group: Advanced
         walrus_metadata_service_name:
           title: walrus_metadata_service_name
           type: string
           description: Walrus metadata service name.
-          default: ''
+          default: ""
           x-walrus-ui:
+            group: Basic
             hidden: true
         walrus_metadata_namespace_name:
           title: walrus_metadata_namespace_name
           type: string
           description: Walrus metadata namespace name.
-          default: ''
+          default: ""
           x-walrus-ui:
+            group: Basic
+            hidden: true
+        context:
```

Comment:
I think we don't need metadata in schema

Reply:
while user-defined schema.yaml existed, will depend on it.